mtp_noms_ops/templates/security/forms/prison-selector-fieldset.html

```diff
@@ -0,0 +1,51 @@
+{% load i18n %}
+
+<div class="heading-large">{% trans 'Prison' %}</div>
+{% with field=form.prison_selector choices=form.prison_selector.field.choices %}
+  <fieldset class="form-group">
+    <legend id="{{ field.id_for_label }}-label" class="visually-hidden">
+      <strong>{{ field.label }}</strong>
+    </legend>
+
+    <div class="{% if field.errors %}form-group-error{% endif %}">
+      {% for value, label in choices %}
+        {% if not user.user_data.prisons and value == form.PRISON_SELECTOR_ALL_PRISONS_CHOICE_VALUE %}
+          {% comment %}User's prisons is already all prisons so we don't want to have two repeated choices.{% endcomment %}
+        {% else %}
+        <div class="multiple-choice">
+          <input id="{{ field.html_name }}-{{ value }}" type="radio" name="{{ field.html_name }}" value="{{ value }}" {% if value == field.value %}checked{% endif %}>
+
+          <label for="{{ field.html_name }}-{{ value }}">
+            {% if value == PRISON_SELECTOR_USER_PRISONS_CHOICE_VALUE %}
+              {% include 'security/forms/user-prison-names-ellipsis.html' %}
+            {% else %}
+              {{ label }}
+            {% endif %}
+          </label>
+
+        </div>
+        {% endif %}
+      {% endfor %}
+
+      {% with sub_field=form.prison %}
+      <div class="mtp-prison-selection-row" id="row_{{ sub_field.name }}">
```

Comment:
what are `div.mtp-prison-selection-row` or `div.mtp-prison-selection-row__prison` used for?

Reply:
> what are div.mtp-prison-selection-row or div.mtp-prison-selection-row__prison used for?

The result of a copy/paste, I'll get rid of them, thanks!